components/add_users_to_team_modal/add_users_to_team_modal.tsx

```diff
@@ -86,7 +86,7 @@ export default class AddUsersToTeamModal extends React.PureComponent<Props, Stat
         let searchResults: UserProfile[] = [];
         const search = term !== '';
         if (search) {
-            const {data} = await this.props.actions.searchProfiles(term, {not_in_team_id: this.props.team.id, replace: true, ...this.state.filterOptions});
+            const {data} = await this.props.actions.searchProfiles(term, {not_in_team_id: this.props.team.id, replace: true, ...Object.fromEntries(this.state.filterOptions)});
```

Comment:
Expansions are tricky. We could expand the map directly, but then we would be expanding the map object, which is not what we want. We would have to be extra careful with these.

Reply:
If we were to use the `{[x: string]: myType | undefined}` approach, we would have also problems, since expanding the object will consider some values undefined.